### PR TITLE
fix java local sdk sample

### PR DESCRIPTION
**Pull request by sushobhit-lt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,80 +64,64 @@ Make sure you have your LambdaTest credentials with you to run test automation s
...
   - For **Windows**:
 
   ```bash
+  set PROJECT_TOKEN="xxxxx#java-sdk-testing"
   set LT_USERNAME="YOUR_USERNAME"
   set LT_ACCESS_KEY="YOUR ACCESS KEY"
   ```
 
-## Run Your First Test
+### Execute SmartUI Java Tests using smartui-cli
```
Comment (JeeveshJ7) [inline README.md:80]: Please change smartui-cli to SmartUI SDK here

File: README.md [hunk 2]
```diff
@@ -64,80 +64,64 @@ Make sure you have your LambdaTest credentials with you to run test automation s
...
 
-> **Test Scenario**: The sample [TestNGTodo1.java](https://github.com/LambdaTest/Java-TestNG-Selenium/blob/master/src/test/java/com/lambdatest/TestNGTodo1.java) tests a sample to-do list app by marking couple items as done, adding a new item to the list and finally displaying the count of pending items as output.
 
-### Configuring Your Test Capabilities
+#### Install the smartui-cli dependencies
```
Comment (JeeveshJ7) [inline README.md:83]: Please change smartui-cli to SmartUI SDK here

File: README.md (hunk 1 above)
Comment (sushobhit-lt) [reply README.md:80]: but we are using cli to execute the test

File: README.md (hunk 2 above)
Comment (sushobhit-lt) [reply README.md:83]: Here we are installing, smartui-cli `npm i @lambdatest/smartui-cli`

File: README.md (hunk 1 above)
Comment (JeeveshJ7) [reply README.md:80]: Okay

**Comment by JeeveshJ7** [comment]
LGTM please merge

**Pull request merged**